src/test/java/uk/ac/ebi/atlas/hcalandingpage/HcaHumanExperimentServiceTest.java

```diff
@@ -27,37 +27,50 @@
     private HcaHumanExperimentService subject;
 
     @Before
-    public void setUp() throws Exception {
+    public void setUp() {
         when(experimentTraderMock.getPublicExperiment(EXPERIMENT_ACCESSION))
                 .thenReturn(MockExperiment.createBaselineExperiment(EXPERIMENT_ACCESSION));
-        when(hcaHumanExperimentDaoMock.fetchExperimentAccessions("organism_part", ""))
-                .thenReturn(ImmutableSet.of(EXPERIMENT_ACCESSION));
-        when(hcaHumanExperimentDaoMock.fetchExperimentAccessions("organism_part", "skin"))
+        subject = new HcaHumanExperimentService(experimentTraderMock, hcaHumanExperimentDaoMock);
+    }
+
+    @Test
+    public void ShouldGetAllHumanExperimentsWithEmptyCharacteristicValue() {
```

Comment:
very minor but the name of the method starts with a lower case letter 🕵️‍♂️

Reply:
Sorry. Silly mistake.